recipes/solarfactors/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "solarfactors" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pvlib/solarfactors/archive/v{{ version }}.tar.gz
+  sha256: 44191ccf45120a8a7457a42e11a0be4503e137c903ab1bb37b56d7d815d3b9be
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION_FOR_SOLARFACTORS={{ version }}
+
```

Is this correct? Isn't it just `SETUPTOOLS_SCM_PRETEND_VERSION`?

```suggestion
    - SETUPTOOLS_SCM_PRETEND_VERSION_FOR_SOLARFACTORS={{ version }}
```